### ISSUE TITLE
Outdated dependency on ESLint

### ISSUE THREAD

**Issue by rbnts** [TrySound/rollup-plugin-eslint#46]
Because `rollup-plugin-eslint` has a dependency on [`eslint^6.0.0`](https://github.com/TrySound/rollup-plugin-eslint/blob/master/package.json#L42), which resolves to `eslint@6.8.0`, running a config based on rules of [`eslint@7`](https://eslint.org/docs/rules/default-case-last) results in unexpected errors like the following.

```
1:1  error  Definition for rule 'default-case-last' was not found  default-case-last
```

<img width="1082" alt="Screenshot 2020-05-12 at 13 56 51" src="https://user-images.githubusercontent.com/6851646/81686488-d6343500-9458-11ea-81b9-bf5e6d734eca.png">

Running the locally installed version of ESLint directly yields the correct output.

<img width="1082" alt="Screenshot 2020-05-12 at 13 57 20" src="https://user-images.githubusercontent.com/6851646/81687310-4347ca80-9459-11ea-9fb4-7b8096183967.png">

**Comment by rbnts** [comment]
Got impatient. Created my own package. https://github.com/robinloeffel/rollup-plugin-eslint
